api.yml

```diff
@@ -185,66 +83,166 @@ paths:
           description: not supported by this implementation!
 components:
   schemas:
-    IssueRequest:
+    ConstructCredentialRequest:
       type: object
+      required:
+        - issuer
+        - vcTemplateReference
+        - claims
+        - output
       properties:
         issuer:
           type: string
-          description: Optional identifier for the issuer of the credential. If this is missing, it may be pre-configured or implied by other values in the request.
-          example: did:example:76e12ec712ebc6f1c221ebfeb1f
-        subject:
+          description: A reference to the issuer's pre-configured profile.
+        vcTemplateReference:
           type: string
-          description: Optional identifier for the subject of the credential. If this is missing, it may be pre-configured or implied by other values in the request, or there may not be an identifier for the subject of the credential.
-          example: did:example:ebfeb1f712ebc6f1c276e12ec21
-        types:
+          description: A reference to the pre-configured Verifiable Credential template.
```

Comment:
What is the format for this template? How does it work? Feels like a spec that would require months of work unto itself.